apps/api/tests/endpoints/test_document_reads.py

```
@@ -0,0 +1,230 @@
+
+class ReadEventIngestTestCase(support.SharedAppTestCase):
+    """The heartbeat sink."""
+
+    def setUp(self) -> None:
+        from imbi.api.auth import permissions
+        from imbi.common import graph, valkey
+
+        self.user = models.User(
+            email='reader@example.com',
+            display_name='Reader',
+            password_hash='$argon2id$hashed',
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "password_hash"

(S106)